docs/user/build-customization.rst

```diff
@@ -247,7 +247,7 @@ This helps ensure that all external links are still valid and readers aren't lin
        python: "3.10"
      jobs:
        pre_build:
-         - python -m sphinx -b linkcheck -D linkcheck_timeout=1 docs/ _build/linkcheck
+         - python -m sphinx -b linkcheck -D linkcheck_timeout=1 docs/ $READTHEDOCS_OUTPUT/linkcheck
```

Comment:
I just wanted to say that I noticed we are using "$READTHEDOCS_OUTPUT" for linkcheck data.. it won't get published and we know this directory is writable. So I think this is a good solution :+1: